src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+
+public class Event extends Task {
+
+    protected String timing;
+
+    public Event(String description, String timing) {
+        super(description);
+        this.timing = timing; 
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + (super.isDone ? "[X]" + super.description : "[ ]"+ super.description) + "(" + timing + ")" ; // mark done task with X
```

Comment:
Could add new lines to improve readability, and comment could be incorporated as part of javadocs

```suggestion
/*
 * @return (what it returns)
 */
        return "[E]"
                + (super.isDone ? "[X]" + super.description : "[ ]"+ super.description) 
                + "(" + timing + ")" ;
```